Sources_v3/APIClient/Endpoints/Payloads/DevicePayloads.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+struct DevicePayload: Decodable, Equatable {
+    private enum CodingKeys: String, CodingKey {
+        case id
+        case createdAt = "created_at"
+    }
+    
+    /// Device identifier.
+    let id: String
+    /// Date the device was created for the user.
+    let createdAt: Date?
```

Comment:
Why do we have this field? It seems it's just a local value anyway, not persisted by the backend?

Reply:
It's actually persisted by backend, but due to some bug it might not be sent. The backend bug exists for a long time, we converted `createdAt` to optional so backend bug doesn't affect end users.